### DISCUSSION TITLE
Can you add other shapes to define or point at a specific area?

### DISCUSSION THREAD

**Discussion by AndrasSCH** [annotorious/annotorious#466] [Q&A]
I was wondering if, besides rectangle and polygon, one could add other shapes, i.e. an arrow or arrowhead to mark an area for annotation. Any advice would be appreciated. Thanks

**Comment by rsimon** [comment]
Dear @AndrasSCH,

the short (but not so good) answer is: no, unfortunately. But it's definitely on my wishlist. (In terms of internal "mechanics", such an annotation would be handled as a point-shape annotation, with a custom marker symbol attached to the point.) I'm sure this will become available at some point. But there are some challenges to be solved first. (I'll reply with a longer & some more context via e-mail.)

**Reply by pranavgoel29** [reply]
Hey @rsimon, I am looking for marking a point x, y with a X or some svg symbol event a React component for more flexibility, is it possible to do this and if yes then how so? Thanks.
